### ISSUE TITLE
Support for 1685-2022

### ISSUE THREAD

**Issue by scottven** [kactus2/kactus2dev#76]
Are there plans for adding support for the new version of IP-XACT?

**Comment by epekkar** [comment]
Yes, the new standard will be supported once we get the update to Kactus2 done.

**Comment by eyalherz** [comment]
Hi, I'm also interested in using Kactus with IP-Xact 2022.
I was on the WG, happy to discuss any questions or issues!
